src/components/dialogs/parameters/shortcircuit/short-circuit-tension-table.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent, useCallback, useMemo } from 'react';
```

Comment:
/**
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

Reply:
Done. Thanks